### ISSUE TITLE
[EN][Spell][Paladin] Holy Wrath

### ISSUE THREAD

**Issue by Wingnanx** [Atlantiss/BugTracker#6923]
http://wowwiki.wikia.com/wiki/Holy_Wrath

Currently holy wrath causes DR with hoj on NON undead targets.

How it should work:
Target = undead -> holy wrath stuns for 3 seconds, if followed by a hoj then hoj will be 3 seconds. 
Target = NOT undead -> holy wrath does not stun, if followed by a hoj then hoj will be 6 seconds.

How it works here:
Target = undead -> holy wrath stuns for 3 seconds, if followed by a hoj then hoj will be 3 seconds. 
Target = NOT undead -> holy wrath does not stun, if followed by a hoj then hoj will be 3 seconds.

The problem is that holy wrath causes stun DR on NON undead targets for no reason.

**Comment by BialyADI** [comment]
It is normal stun have Dr with Stun simple.
If DK use Lish Born you want him stun for 9sec ;D?
No you stun him max for 7,5sec, Hoj and Wrath

**Comment by Prodiege** [comment]
All of the comments on the link are from 5.XX or newer, where is any proof? Also what happens if Holy Wrath is glyphed?

**Comment by Wingnanx** [comment]
@BialyADI Did you bother reading the topic, 
I clearly stated how it should work (if you use holy wrath first 3 seconds on undead target, followed by hoj another 3 seconds because its DR total of 6 seconds CC)

Now what im asking is NOT to stop holy wrath from applying DR to UNDEAD targets. I'm asking to remove the part where it DR's NON UNDEAD targets. Because as I stated if you use holy wrath on NON undead targets it applies DR while it should only apply DR when it actually stuns the target (which only happens on undead targets).

@Prodiege I changed the link: http://wowwiki.wikia.com/wiki/Holy_Wrath
I quote: "We just deployed a hotfix to modify Holy Wrath so this spell has the same diminishing return mechanics that other player stun spells and abilities currently have." Which hasn't changed so that means if a target is stunned, then it should apply DR for stuns. But only if it stuns, think of using poly 4 times in a row, first it will be 8 seconds, then it will be 4 seconds, then it will be 2 seconds, the 4th time it will not apply and will also not start the timer for DR on it. This is the same thing if a target is not stunned by the effect... it should not apply a stun DR if it does not stun the target.

Glyphed it should obviously react the same as it does to undead, but not also to elementals and dragonkin, if the target gets stunned by holy wrath, then it should apply DR and for example hoj should be 3 seconds / DR.
